### DISCUSSION TITLE
PWR Automate 승인 관련 에러

### DISCUSSION THREAD

**Discussion by briankim1027** [fusiondevkr/pl900#39] [1 Q&A]
![승인 완료 메시지](https://user-images.githubusercontent.com/34652528/128723878-8a2ef85e-1240-4323-be5d-865ab6d2cf80.jpg)
제 개발자 환경에서 트위터 프로그램 테스트 하려고 하니
첨부한 파일과 같이 그 환경에서 PWR Automate 승인 권한이 없다고 나오는데
어찌해야 하는지요? 

테스트 해보니 승인완료까지 다 되기는 합니다.
그리고 제가 개발자 default 환경을 brian1027@min8896.onmicrosoft.com 요렇게 만들었는데 LNU FNU로 되어 있는 것도 좀...
요것도 확인 가능한가요?

![메시지](https://user-images.githubusercontent.com/34652528/128722912-3ade9c98-dd93-49a5-8267-023cd1742b1d.jpg)

**Comment by sesme100** [comment]
Default 환경 이름 내용 관련하여 하기 참조 부탁드립니다. 
https://github.com/fusiondevkr/pl900/discussions/27

**Comment by sesme100** [comment]
에러가 똑같이 나도 승인 흐름을 그대로 실행하여 프로비저닝 하시면 몇 분 후 만드신 흐름에 대하여 정상 작동 할 것입니다. 
(저도 똑같은 에러가 나왔는데 에러 설명 그대로 승인 흐름을 실행하니 정상 작동 하였습니다.  )
